agent/connect/ca/testing.go

```diff
@@ -2,6 +2,7 @@ package ca
 
 import (
 	"fmt"
+	"github.com/stretchr/testify/require"
```

Comment:
So `testing.go` files are technically compiled into the non-test code for the package (`agent/connect/ca`) so since testify uses `"testing"` from the stdlib, this adds test plumbing to the compiled consul binary which we try to avoid.

It's easy in this case to switch `require.Equal(t, a, b)` to
```
if a != b {
  t.Fatalf("err: %v != %v", a, b)
  }
  ```